### PR TITLE
LPS-99963 Display test status and details in the sidebar

### PR DESCRIPTION
**Pull request by dgarciasarai**
Hey @bryceosterhaus ,

Can you review from https://github.com/bryceosterhaus/liferay-portal/commit/1f5b85cfe2c3da4463f5dfec8d9898bde78e071d to https://github.com/bryceosterhaus/liferay-portal/commit/5470ddbd084072e3c4df9afeaeb59c9b4954d239?
The other commits are already in Brian's master branch (from https://github.com/bryceosterhaus/liferay-portal/commit/298c961de83fe0d9f50ff258acda74fd81914908 to https://github.com/bryceosterhaus/liferay-portal/commit/1b0777ca69cf84fe95638d9c0a533d4d0ec2558c).
Thanks.

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

File: modules/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/Variants/Variants.es.js [hunk 1]
```diff
@@ -31,11 +31,36 @@ function Variants({
...
 
 	return (
-		<div className="mt-3">
-			<h4 className="sheet-subtitle">
+		<>
+			<h4 className="mb-3 mt-4 sheet-subtitle">
 				{Liferay.Language.get('variants')}
 			</h4>
 
+			{variants.length === 1 ? (
```
Comment (bryceosterhaus) [inline modules/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/Variants/Variants.es.js]: No need to do a ternary, using a simple `&&` should be fine.

```jsx
{someBoolean && 
    //...
}

**Comment by dgarciasarai** [comment]
Tests running here https://github.com/dgarciasarai/liferay-portal/pull/100

**Comment by bryceosterhaus** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/77284. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**